source/exploring_catalog_and_datasets/02_looking_up_a_dataset/filtering_dataset.rst

```diff
@@ -0,0 +1,45 @@
+Filtering a dataset
+===================
+
+When looking up a dataset, as a user on an open data portal, you may be looking for specific data from the dataset. To find that specific data more easily, instead of scrolling through dozens or hundreds of records, different filtering options are available. The filtering options are located on the left side of the interface.
+
+.. image:: images/filtering_dataset.png
+   :alt: Dataset filters
+
+It is possible to filter:
+
+- by textual research, using the search bar
+- by existing value, using the available field-based filters
+
+Using the search bar
+--------------------
+
+The search bar should be used to make textual searches in the dataset. When doing a textual search, the platform will look for a match between the search and the records. The dataset will be filtered to only display those matching records.
+
+To make a textual search:
+
+1. Click on the search bar area.
+2. Type the word to search in the dataset.
+3. Press enter for the search to be taken into account, and the dataset filtered according to the typed word.
```

Comment:
> 3. Press enter for the search to be taken into account, and the dataset filtered according to the typed word.

Maybe simplified by: 
"Press enter to start the search and filter the dataset according to the typed word."